experimental/dds/tree2/src/feature-libraries/editable-tree-2/lazyField.ts

```diff
@@ -224,6 +224,14 @@ export abstract class LazyField<TKind extends FieldKind, TTypes extends AllowedT
 	}
 }
 
+function assertIsLazySequence<TTypesSource extends AllowedTypes>(
+	sourceField: unknown,
+): sourceField is LazySequence<TTypesSource> {
```

Comment:
```suggestion
): sourceField asserts LazySequence<TTypesSource> {
```
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions